packages/patternfly-4/react-core/src/components/FormSelect/FormSelect.tsx

```diff
@@ -48,14 +51,15 @@ export class FormSelect extends React.Component<FormSelectProps> {
   };
 
   render() {
-    const { children, className, value, isValid, isDisabled, ...props } = this.props;
+    const { children, className, value, isValid, isDisabled, isRequired, ...props } = this.props;
     return (
       <select
         {...props}
```

Comment:
We should move `{...props}` to the end to not break consumers using the `required` property instead.